src/auth/interfaces/accessToken.ts

```diff
@@ -1,3 +1,8 @@
 export interface AccessToken {
-    accessToken: string;
-}
\ No newline at end of file
+  accessToken: string;
+}
+
+export interface UserLogin {
```

Comment:
What is this interface doing?

Reply:
if is used in the login service as asked by some frontend devs to send back not only the token but also the isAdmin 
```js
async login(credentials: LoginUserDto): Promise<UserLogin> {
    const user = await this.usersService.findOneUnsanitized({ email: credentials.email });
    const passwordHash = user.password;
    const isCorrectPassword = await bcrypt.compare(credentials.password, passwordHash);
    if (!isCorrectPassword || !passwordHash) {
      throw new UnauthorizedException();
    }

    // Add a username in the JWT token
    const payload = { username: user.username, isAdmin: user.isAdmin, id: user.id };

    return { accessToken: await this.jwtService.signAsync(payload), isAdmin: user.isAdmin };